test/remote/db_tests.js

```diff
@@ -272,35 +284,13 @@ describe('remote db', function() {
         .then(sessions => {
           assert.equal(sessions.length, 1, 'sessions still contains one item')
           assert.equal(sessions[0].uaBrowser, 'Firefox Mobile', 'uaBrowser property is correct')
-          assert.equal(sessions[0].uaBrowserVersion, '41', 'uaBrowserVersion property is correct')
+          assert.equal(sessions[0].uaBrowserVersion, '42', 'uaBrowserVersion property is correct')
           assert.equal(sessions[0].uaOS, 'Android', 'uaOS property is correct')
           assert.equal(sessions[0].uaOSVersion, '4.4', 'uaOSVersion property is correct')
           assert.equal(sessions[0].uaDeviceType, 'mobile', 'uaDeviceType property is correct')
           assert.equal(sessions[0].uaFormFactor, null, 'uaFormFactor property is correct')
           assert.equal(sessions[0].location, null, 'location property is correct')
 
-          // Fetch the session token
-          return db.sessionToken(tokenId)
-        })
-        .then(sessionToken => {
-          // Update the session token
-          return db.updateSessionToken(sessionToken, 'Mozilla/5.0 (iPad; CPU iPhone OS 8_3 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) FxiOS/1.0 Mobile/12F69 Safari/600.1.4', '1.1.1.1')
```

Comment:
I only added this block recently, as extra coverage for updating the `sessionToken` object inside `db.updateSessionToken`. Now that it doesn't do that any more, the extra assertions are redundant.